frontend/src/app/api/metaData.ts

```diff
@@ -0,0 +1,10 @@
+import { baseAPIurl } from './config';
+
+export async function fetchMetadataHash(url: string) {
+    const res = await fetch(`${baseAPIurl}/metadata?metadata_url=${url}`);
```

Comment:
use encodeURIComponent function just in case.

Reply:
Done